pom.xml

```diff
@@ -11,6 +11,13 @@
 
 	<packaging>pom</packaging>
 
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>3.1.4</version>
```

Comment:
```suggestion
		<version>${spring-boot.version}</version>
```

Reply:
No, this will not work because the parent `pom.xml` is resolved first before evaluating properties, so, let's leave it as is.